### PR TITLE
Add passwd file gatherer

### PR DESCRIPTION
**Pull request by arbulu89**
`/etc/passwd` file gatherer.
It returns a list for each found entry with struct defined as `Entry`

### CODE REVIEW

File: internal/factsengine/gatherers/passwd.go [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+func NewPasswdGatherer(path string) *PasswdGatherer {
+	return &PasswdGatherer{
+		passwdFilePath: path,
+	}
+}
+
+func (g *PasswdGatherer) Gather(factsRequests []entities.FactRequest) ([]entities.Fact, error) {
+	facts := []entities.Fact{}
+	log.Infof("Starting %s state facts gathering process", PasswdGathererName)
```
Comment (rtorrero) [inline internal/factsengine/gatherers/passwd.go:58]: What does the word `state` reflect here? I'd use "file"

**Review by rtorrero — APPROVED** [review]
LGTM, just a minor comment

File: internal/factsengine/gatherers/passwd.go (hunk 1 above)
Comment (arbulu89) [reply internal/factsengine/gatherers/passwd.go:58]: arggg
copy/paste typo

**Review by CDimonaco — APPROVED** [review]
LGTM

**Pull request merged**